layouts/shortcodes/qbo/latest-version.html

```diff
@@ -0,0 +1 @@
+1.8.0
```

Comment:
[Your guide](https://docs.qbo.io/#/qke?id=kubeflow) only explain how to use `kubeflow/manifests` for `1.7.0`:

```suggestion
1.7.0
```

Reply:
qbo has been tested with version 1.8.0 using the same guide. 

Prerequisites and dependencies for 1.8.0 are listed here:

https://docs.qbo.io/#/qke?id=kubeflow-v180-with-nvidia-gpu-support

qbot which is the `bot` doing the automatic deployment for kubeflow has CLI support for kubeflow 1.8.0 

https://github.com/alexeadem/qbot/blob/eee049f544865395a83f9a616915a68581e518aa/demos/kubeflow/qbot#L69

and pulling raw 1.8.0 manifest from upstream

https://github.com/alexeadem/qbot/blob/eee049f544865395a83f9a616915a68581e518aa/demos/kubeflow/qbot#L149